cryptlex/cryptlex-enterprise/templates/secrets/web-portals-secret.yaml

```diff
@@ -3,7 +3,3 @@ kind: Secret
 metadata:
   name: {{ .Release.Name }}-web-portals-secret
 type: Opaque
-stringData:
-{{ if .Values.webPortals.customDomainAccessToken }}
-  CUSTOM_DOMAIN_ACCESS_TOKEN: {{ .Values.webPortals.customDomainAccessToken  | quote }}
```

Comment:
@mumin-khan don't we sue this?

Reply:
@adnan-kamili As far as I know, this was used to hit /admin/accounts endpoint and since we now fetch config from config endpoint, we no longer need it.

Reply:
@mumin-khan this is needed till customers migrate to new custom domains

Reply:
@adnan-kamili but we no where use it in the web-portals.

Reply:
@adnan-kamili ?

Reply:
@mumin-khan this is used in the legacy web portal to support branding

Reply:
@adnan-kamili legacy web-portal is dashboard and has a seperate secret file ( in fact everything is seperate for it ).

Reply:
ok, please incrememt the patch version in chart.yaml for both versions

Reply:
secrets file can be delete an dits reference removed as it doesn't contain any datat